datamade/erikson-edi
# adds tools-faq and press pages

## Overview

Adds Tools/FAQ and Press page stubs

Closes #150 

## Testing Instructions

* Run the app locally and note the two new menu items under the Resources dropdown
* Ensure the pages load and show the correct titles (content is TBD)
